From 9318f4a5cefffc0e33e635e92f0df27a130b6cd3 Mon Sep 17 00:00:00 2001
From: Dave Taht <d+bufferbloat@taht.net>
Date: Sun, 24 Jul 2011 22:23:33 -0700
Subject: [PATCH 09/13] correct links for iptables ip6tables to xtables

---
 package/iptables/Makefile |   14 +++++++-------
 1 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/package/iptables/Makefile b/package/iptables/Makefile
index a0ab076..dc4ec76 100644
--- a/package/iptables/Makefile
+++ b/package/iptables/Makefile
@@ -10,7 +10,7 @@ include $(INCLUDE_DIR)/kernel.mk
 
 PKG_NAME:=iptables
 PKG_VERSION:=1.4.12
-PKG_RELEASE:=3
+PKG_RELEASE:=4
 
 PKG_MD5SUM:=d3f145c2c91daecbb4251bc79390b46c
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
@@ -450,9 +450,9 @@ endef
 
 define Package/iptables-utils/install
 	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/iptables-multi $(1)/usr/sbin/
-	$(LN) iptables-multi $(1)/usr/sbin/iptables-save
-	$(LN) iptables-multi $(1)/usr/sbin/iptables-restore
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/xtables-multi $(1)/usr/sbin/
+	$(LN) xtables-multi $(1)/usr/sbin/iptables-save
+	$(LN) xtables-multi $(1)/usr/sbin/iptables-restore
 endef
 
 define Package/ip6tables/install
@@ -466,9 +466,9 @@ endef
 
 define Package/ip6tables-utils/install
 	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/ip6tables-multi $(1)/usr/sbin/
-	$(LN) ip6tables-multi $(1)/usr/sbin/ip6tables-save
-	$(LN) ip6tables-multi $(1)/usr/sbin/ip6tables-restore
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/xtables-multi $(1)/usr/sbin/
+	$(LN) xtables-multi $(1)/usr/sbin/ip6tables-save
+	$(LN) xtables-multi $(1)/usr/sbin/ip6tables-restore
 endef
 
 define Package/libiptc/install
-- 
1.7.4.1

